src/Sylius/Component/User/Context/CustomerContextInterface.php

```diff
@@ -19,7 +19,7 @@
 interface CustomerContextInterface
 {
     /**
-     * @return CustomerInterface|null
+     * @return null|CustomerInterface
```

Comment:
Anyway, it returns `null` first (in code), because it's the way we handle *edge* cases, we use early returns. The main target of `getCustomer()` method is to get customer.